### PR TITLE
Feat: Category

### PR DESCRIPTION
**Pull request by pritam-kr**
- Added category at home page
- change category filter logic

### CODE REVIEW

File: src/Component/Hero/Hero.css [hunk 1]
```diff
@@ -62,3 +62,22 @@ color: var(--white-text);
...
+
+/*=========Category chips in Home Page============*/
+.videos-category{
+  margin: 1rem 0;
+}
+.category-chips{
+  
+  backdrop-filter: blur(10px) saturate(180%);
+  -webkit-backdrop-filter: blur(5px) saturate(180%);
+  background-color: #2ea4da49;
```
Comment (AbhishekkGautam) [inline src/Component/Hero/Hero.css:76]: we should use CSS variable for color.

**Review by AbhishekkGautam — APPROVED** [review]
everything works as expected. good to merge

File: src/Component/Hero/Hero.jsx [hunk 2]
```diff
@@ -1,27 +1,62 @@
...
-                <p className="text-lg" style={{ marginTop: "1rem" }}>PlayMe is a learning platform for students.</p>
-                <div className="searchbar-wrapper center">
-                    <button className="btn btn-explore center" onClick={() => navigate("/explore")}>
-                        <BiIcons.BiVideoPlus className="icons video-icon" />Explore </button>
-                </div>
-                <FaIcons.FaGithub />
-            </div>
+        <div className="videos-category">
+          <h2 className="text-md">Videos Categories</h2>
+          {getUniqueCategory?.map((eachCategory, i) => (
```
Comment (vijaytembugade) [inline src/Component/Hero/Hero.jsx:45]: This is really Nice. `Optional chaining` parameter !

File: src/Component/Hero/Hero.jsx [hunk 3]
```diff
@@ -1,27 +1,62 @@
-import React from "react"
-import "./Hero.css"
+import React from "react";
+import "./Hero.css";
 import * as FaIcons from "react-icons/fa";
-import {useNavigate} from "react-router-dom"
+import { useNavigate } from "react-router-dom";
 import * as BiIcons from "react-icons/bi";
- 
+import { useVideoContext, useAuthContext } from "../../Context/Index";
```
Comment (msdeshmukh009) [inline src/Component/Hero/Hero.jsx:6]: You can write path as `../../context` no need to write `Index`.

**Review by vijaytembugade — APPROVED** [review]
Everything lookks good. Clean UI.
image

**Review by msdeshmukh009 — APPROVED** [review]
The code looks clean and UI is working as expected. Good work!!

**Pull request merged**